TimePlatformer/Assets/Scripts/PauseMenuScript.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PauseMenuScript : MonoBehaviour {
+
+
+    public void pause(){
+        Debug.Log("pausing");
+        Time.timeScale = 0;
+        var continueButton = GameObject.FindGameObjectWithTag("ContinueButton");
```

Comment:
Please use the explicit types for variables. Also please move all of these handles to a `public void Start()` function so that you aren't grabbing them at more than once.